packages/ckeditor5-theme-lark/theme/ckeditor5-table/tableediting.css

```diff
@@ -14,13 +14,16 @@
 		&.ck-editor__nested-editable:focus {
 			/* A very slight background to highlight the focused cell */
 			background: var(--ck-color-selector-focused-cell-background);
-
-			/* Fixes the problem where surrounding cells cover the focused cell's border.
-			It does not fix the problem in all places but the UX is improved.
-			See https://github.com/ckeditor/ckeditor5-table/issues/29. */
-			border-style: none;
 			outline: 1px solid var(--ck-color-focus-border);
 			outline-offset: -1px; /* progressive enhancement - no IE support */
+
+			/*
+			 * Fixes issue with the hidden border of the last focused cell during selection.
+			 * Keep it in sync with the border color of the `.ck-content .table td, th` styling.
+			.*
+			 * See more: https://github.com/ckeditor/ckeditor5/issues/16979
+			 */
+			border: 1px solid hsl(0, 0%, 75%);
```

Comment:
What if a user changes this default style to some other? We are hardcoding the style that could be different in some user's integration in my opinion, but I'm not 100% sure.

Reply:
It may be, but I did not come up with a better solution.